src/app/components/Navbar/Navbar.jsx

```diff
@@ -97,7 +97,7 @@ export default function App({ featureRef }) {
             as={Link}
             href="/login"
```

Comment:
```suggestion
            href="/join"
```

Same Here ! Change the href!